wlauto/common/android/workload.py

```diff
@@ -262,12 +278,13 @@ def reset(self, context):  # pylint: disable=W0613
 
         # As of android API level 23, apps can request permissions at runtime,
         # this will grant all of them so requests do not pop up when running the app
+        # This can also be done less "manually" during adb install using the -g flag
         if self.device.get_sdk_version() >= 23:
             self._grant_requested_permissions()
```

Comment:
this no longer needs to be done (the method can be removed entirely) since the -g option for adb install handles this now